concordia/settings_prod.py

```diff
@@ -78,3 +78,5 @@
 REGISTRATION_SALT = "django_registration"  # doesn't need to be secret
 
 SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
+
+RATELIMIT_BLOCK = os.getenv("RATELIMIT_BLOCK", True).lower() not in ("false", "0")
```

Comment:
The default value needs to be a type which implements `.lower()`:

```suggestion
RATELIMIT_BLOCK = os.getenv("RATELIMIT_BLOCK", "").lower() not in ("false", "0")
```